packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceOpenDocumentation.test.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import { createSandbox, SinonSandbox, SinonStub } from 'sinon';
+import * as vscode from 'vscode';
+import {
+  apexDocUrl,
+  auraDocUrl,
+  defaultDocUrl,
+  forceOpenDocumentation,
+  functionsDocUrl,
+  lwcDocUrl,
+  soqlDocUrl
+} from '../../../src/commands';
+
+describe('forceOpenDocumentation', () => {
+  let sb: SinonSandbox;
+  let activeTextEditorStub: SinonStub;
+  let openExternalStub: SinonStub;
+
+  beforeEach(() => {
+    sb = createSandbox();
```

Comment:
This should go outside the before each I think https://sinonjs.org/releases/latest/sandbox/

Reply:
That's not what we have in our code.  Most(and I thought all) of our test code have the createSandbox() call in the beforeEach().  Searching the codebase, I found...
packages/salesforcedx-utils-vscode/test/unit/commands/channelService.test.ts
packages/salesforcedx-utils-vscode/test/unit/commands/commandletExecutor.test.ts
packages/salesforcedx-utils-vscode/test/unit/helpers/traceFlags.test.ts
.
.
.

There are some with it outside the beforeEach(), like in packages/salesforcedx-utils-vscode/test/unit/helpers/paths.test.ts, but the majority of our code has it in there.

Reply:
Yup, I think it's wrong in most of our code :)

Reply:
done